assets/queries/cloudFormation/ec2_network_acl_duplicate_rule/query.rego

```diff
@@ -38,3 +38,27 @@ getTraffic(entry) = "egress" {
 } else = "ingress" {
 	true
 }
+
+compareRuleNumber(entry1, entry2){
```

Comment:
Couldn't this be simplified to `to_number` like other changes?

Reply:
You're absolutely right! I missed that one, thanks for the heads up.